src/tools/imgtool/modules/os9.cpp

```diff
@@ -44,7 +44,7 @@ struct os9_diskinfo
 	unsigned int octal_track_density : 1;
 
 	char name[33];
-	uint8_t *allocation_bitmap;
+	uint8_t allocation_bitmap[65536];
```

Comment:
Does anything create an `os9_diskinfo` on the stack?  If something does, it’s probably better to make this a `vector` or `unique_ptr`.

Reply:
I verified that nothing creates one on the stack. I totally would have used allocated storage except that the archaic architecture of imgtool allocates raw byte buffers to hold this data, so new/delete are never called. The whole allocation strategy needs a rethink longer term. This should be sufficient for now. It's only 64k.